roofit/roofitcore/inc/RooAbsPdf.h

```diff
@@ -353,6 +353,13 @@ class RooAbsPdf : public RooAbsReal {
   
   TString _normRange ; // Normalization range
   static TString _normRangeOverride ; 
+
```

Comment:
Should we have these functions in RooAbsPdf, that is already a big class ?
Are maybe more functions of RooFitResult or  some other helper classes  ?
Do they need to use some private RooAbsPdf data or functions ? 

If they need to be member function of RooAbsPdf I would not use the world "set" because they are not setting anything in the class, but something like:
`calculateAsymptoticCorrectedCovMatrix`

Reply:
That's true, they don't use any private RooAbsPdf data or functions so the new functions could also go somewhere else.

However, the point of these new functions was not to refactor RooAbsPdf for the sake of making the class less big, but it's reducing code duplication such that the definition of the corrected covariance matrix qualities don't have to be maintained at two separate places. I changed the names as you suggested, but I think we should leave further refactoring for other PRs.